plugins/pulp_rpm/plugins/importers/yum/upload.py

```diff
@@ -179,42 +181,56 @@ def _handle_erratum(repo, type_id, unit_key, metadata, file_path, conduit, confi
     unit.save()
 
     if not config.get_boolean(CONFIG_SKIP_ERRATUM_LINK):
+        rpm_filter_list = []
+        for search_dict in unit.rpm_search_dicts:
+            # TODO should we del checksum and checksumtype from search_dict
```

Comment:
I don't think a rpm and an errata for that rpm would have the same `checksum` or `checksumtype`.